novatel_gps_driver/src/nodelets/novatel_gps_nodelet.cpp

```diff
@@ -356,6 +369,51 @@ namespace novatel_gps_driver
     }
 
     /**
+        * SystemAlert to shutdown the sensor node
+        */
+      void alertCallback(const cav_msgs::SystemAlertConstPtr &msg)
+       {
+         if( msg->type==cav_msgs::SystemAlert::FATAL || msg->type==cav_msgs::SystemAlert::SHUTDOWN)
+           {
+            ros::shutdown();
+           }
+       }
+
+      /**
+          * DriverStatus function definition various novatel driver status
+          */
+      void publish_status(const ros::TimerEvent&)
+      {  // Set driver type
+          status_.gps=true;
+          status_.imu=true;
+
+          time_difference=ros::Time::now()-last_update_time_;
+
+          if (last_update_time_.isZero() || ((time_difference>ros::Duration(2.0)) && (status_gps==cav_msgs::DriverStatus::OPERATIONAL)))
+          {
+            status_gps=cav_msgs::DriverStatus::OFF;
+          }
+
+          if (status_gps==cav_msgs::DriverStatus::OFF)
+          {
+           status_.status=cav_msgs::DriverStatus::OFF;
+          }
+          else if (status_gps==cav_msgs::DriverStatus::OPERATIONAL)
+          {
+          status_.status=cav_msgs::DriverStatus::OPERATIONAL;
+          }
+          else if (status_gps==cav_msgs::DriverStatus::FAULT)
+          {
+          status_.status=cav_msgs::DriverStatus::FAULT;
+          }
+          else if(status_gps==cav_msgs::DriverStatus::DEGRADED)
+          {
+           status_.status=cav_msgs::DriverStatus::DEGRADED;
+          }
```

Comment:
These assignment if statements are redundant. 
Since you are comparing with a uint anyway why not just assign it directly?
status_.status = status_gps